ccd/deployment.yaml

```diff
@@ -0,0 +1,825 @@
+---
+#
+# NAMESPACE
+#
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ccd
+---
+#
+# IDAM-AUTHENTICATION-WEB CONFIG
+#
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: idam-authentication-web
```

Comment:
We should be able to use the AAT (tactical preprod for now) version of idam and not need to run this.
Unless there's a good reason to bring everything into AKS?